### DISCUSSION TITLE
Alphabetical Sorting Base Models

### DISCUSSION THREAD

**Discussion by alkaris2** [open-webui/open-webui#7462] [Ideas]
# Feature Request

When selecting **Base Models (From)** drop-down list, it's a jumbled mess which makes searching for the **Base Models (From)** It would be nice if these can be alphabetically sorted in the list, instead of an out of order jumbled mess. Also adding a Search function so you can just type the name of the Base Model and have it instantly come up after typing the first 3 characters.

## Important Notes

- Alphanumerical Sorting of model lists in Dropdown list.
- Search Function from typing the first 3 characters of a **Base Model** name

---

**Is your feature request related to a problem? Please describe.**
Searching for a Base Model to choose from the dropdown list is disorganized and hard to search through. While I can type a letter of a name of a model by pressing a single letter on the keys to cycle through them that begins with that first key character, it doesn't let me full properly search to the exact one I want to select from.

**Describe the solution you'd like**
- Sorting of Base Model (From) lists in [A-Z | a-z | 0-9] ordered sorting.

**Additional context**

I'm quite surprised this hasn't already been implemented as a default, or that I would need to Request that as a Feature.

**Comment by tajhlande** [comment]
Is this still an open discussion?  I'm a bit surprised. I know the admin can change the order of models in the display, but even there, I would expect a "sort alphabetically", "sort by download date", something...
